src/components/Description/index.tsx

```diff
@@ -8,11 +8,27 @@ import Paragraph from './Paragraph';
 import cssBinder from 'styles/cssBinder';
 
 import styles from './style.css';
+import fonts from 'EBI-Icon-fonts/fonts.css';
+import Tooltip from '../SimpleCommonComponents/Tooltip';
 
-const css = cssBinder(styles);
+const css = cssBinder(styles, fonts);
 
+export const getDescriptionText = (
+  description: string | StructuredDescription,
+) => (typeof description === 'string' ? description : description.text);
+
+export const hasLLMParagraphs = (
+  description: Array<string | StructuredDescription>,
+): boolean => {
+  if (!description?.length) return false;
+  if (typeof description?.[0] === 'string') return false;
+  const llmDescrpitions = description.filter(
+    (desc) => (desc as StructuredDescription).llm,
+  );
+  return !!llmDescrpitions.length;
```

Comment:
Readability?

```suggestion
  return llmDescrpitions.length > 0;
```

 (Well, more a personal preference tbh)